src/System.Runtime.Handles/src/project.json

```diff
@@ -5,6 +5,11 @@
             "Microsoft.TargetingPack.Private.CoreCLR": "1.0.0-rc2-23520"
         }
     },
+    "netcore50":{
+      "dependencies": {
+            "Microsoft.TargetingPack.Private.NetNative": "1.0.0-rc2-23607"
```

Comment:
Please make the Private.CoreCLR and Private.NETNative versions match for consistency. Also remove the pre-release version of .NETFramework\v4.6 package.